site/content/blog/2021-02-10-automate-checking-asvs-controls-using-zap-scripts/index.md

```diff
@@ -0,0 +1,326 @@
+---
+title: "Automate checking ASVS controls using ZAP scripts"
+description: "Automate checking ASVS controls using ZAP scripts"
+summary: "Write scripts in ZAP which will check a target application's compliance against ASVS controls."
+type: post
+tags:
+- blog
+date: "2021-02-10"
+addSocialPreview: true
```

Comment:
false unless you have a good image to include :)

Reply:
I've just asked in slack if ASVS has a logo (I searched quickly and didn't see one).